index.html

```diff
@@ -1820,9 +1820,11 @@ <h2>Chunk types</h2>
   <li>Colour space information: <a href="#chrm-primary-chromaticities-and-white-point"><span class=
   "chunk">cHRM</span></a>, <a href="#gama-image-gamma"><span class=
   "chunk">gAMA</span></a>, <a href="#iccp-embedded-icc-profile"><span class=
-  "chunk">iCCP</span></a>, <a href="#sbit-significant-bits"><span class=
+  "chunk">iCCP</span></a>, <a href="#iccn-embedded-icc-profile"><span class=
+  "chunk">iCCN</span></a>, <a href="#sbit-significant-bits"><span class=
```

Comment:
As discussed on the call today, all instances of `iCCN` should now be changed to `iCCM` and similarly, to not break links, change all "iccn-embedded-icc-profile" to "iccm-embedded-icc-profile"